src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -327,3 +316,38 @@ def get_config_actions(self) -> List[Tuple[str, str]]:
                 ),
             ),
         ]
+
+    @wrap_api_errors
+    def pre_register_submission(self, submission: "Submission", options: dict) -> None:
```

Comment:
Same as StUF-ZDS - let's define this method above the main registration method for more natural reading.